### PR TITLE
Implement tdbg AdminDeleteWorkflow

### PR DESCRIPTION
**Pull request by yycptt**
**What changed?**
- Implement tdbg AdminDeleteWorkflow

**Why?**
- Implement tdbg AdminDeleteWorkflow

**How did you test it?**
- Tested locally

**Potential risks**

**Is hotfix candidate?**
- No.

### CODE REVIEW

File: tools/tdbg/util.go [hunk 1]
```diff
@@ -63,10 +63,10 @@ func prettyPrintJSONObject(o interface{}) {
 	fmt.Println()
 }
 
-func getRequiredGlobalOption(c *cli.Context, optionName string) (string, error) {
+func getRequiredOption(c *cli.Context, optionName string) (string, error) {
 	value := c.String(optionName)
```
Comment (yycptt) [inline tools/tdbg/util.go:67]: Looks like in cli v2 library, there's no `GlobalString` method and `String` can be used to get both local and global flags. So only one version of get required option is needed for both local and global flags.

File: tools/tdbg/commands.go [hunk 2]
```diff
@@ -180,9 +180,51 @@ func describeMutableState(c *cli.Context) (*adminservice.DescribeMutableStateRes
 	return resp, nil
 }
 
-// AdminDeleteWorkflow delete a workflow execution from Cassandra and visibility document from Elasticsearch.
+// AdminDeleteWorkflow force delete a workflow's mutable state, history, and visibility records as long as it's possible.
```
Comment (yux0) [inline tools/tdbg/commands.go:183]: also current record?

File: tools/tdbg/commands.go [hunk 3]
```diff
@@ -180,9 +180,51 @@ func describeMutableState(c *cli.Context) (*adminservice.DescribeMutableStateRes
...
+
+	namespace, err := getRequiredOption(c, FlagNamespace)
+	if err != nil {
+		return err
+	}
+	wid, err := getRequiredOption(c, FlagWorkflowID)
+	if err != nil {
+		return err
+	}
+	rid := c.String(FlagRunID)
```
Comment (yux0) [inline tools/tdbg/commands.go:198]: Should RunID also be required in this case?

**Review by yux0 — APPROVED** [review]

File: tools/tdbg/commands.go (hunk 3 above)
Comment (yycptt) [reply tools/tdbg/commands.go:199]: hmm I think we should support operate only based on workflowID to be consistent with all other commands. Could be useful if only workflowID is known or operator knows the target workflow is open. 

There's a confirmation step so I am not quite worry about operator forgets to specify the runID and accidentally deleted the current run.

File: tools/tdbg/commands.go (hunk 2 above)
Comment (yycptt) [reply tools/tdbg/commands.go]: Updated comment.

**Pull request merged**